kiarash/TS/TS-declaration-files/src/myModule.ts

```diff
@@ -0,0 +1,12 @@
+export const foo = () => (Math.random() > 0.5 ? "hello" : 5);
```

Comment:
Writing declaration files is pointless if we have TS files. This should be a JS file.